### ISSUE TITLE
"Copy to clipboard as PNG" - dark mode does not work

### ISSUE THREAD

**Issue by simonmeggle** [zsviczian/obsidian-excalidraw-plugin#590]
Hello, 

I am on the newest version of obsidian-excalidraw and need to export a drawing to the clipboard. 
The Obsidian theme is dark. 

I cannot get the "copy to clipboard as PNG" to use the dark mode; the clipboard always contains a light version of the image. 
If I, however, use "Save as PNG into Vault", dark mode is used as expected. 
Did I miss something to get the same behavior with the clipboard or is this a bug? 
Thanks for this great plugin!
Simon

**Comment by KsanaE-0** [comment]
Same issue here.
- Excalidraw Version: 1.8.6
- Obsidian Version: v1.1.6
- MacOS Version: Ventura 13.2


![2022-12-18_21-24-06 (1)](https://user-images.githubusercontent.com/24473107/208317737-0ca84446-4fc8-433c-a3dd-0292d201a629.gif)

**Comment by neito5** [comment]
Same problem with the latest versions of both Obsidian and the plugin on Windows 10. According to [excalidraw.com Issue 5023](https://github.com/excalidraw/excalidraw/issues/5023): **_"that action (and any export action) uses the export theme setting (you can set in the export dialog.)"_** - which is not available in the plugin.

"Copy to clipboard as SVG" does nothing at all.

**Comment by Zebrazilla** [comment]
Same issue here.

**Comment by zsviczian** [comment]
This is a possible workaround... Try "Export Image" from the "hamburger menu" instead of using the context menu. However.... copy to cliboard in the hamburger menu is currently only supported on Obsidian desktop (i.e. not mobile like iPad, tablet, etc.)

![1896CAC6-B75D-4FCB-B71B-F51188301151](https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/14358394/c419d9a8-0abb-4064-aa47-8221f56a5319)

**Comment by simonmeggle** [comment]
Yes.... that is a workaround. (Are you planning to fix that issue?) 
Thanks a lot!

**Comment by zsviczian** [comment]
Does it work on Excalidraw.com?

**Comment by amusingaxl** [comment]
> This is a possible workaround... Try "Export Image" from the "hamburger menu" instead of using the context menu. However.... copy to cliboard in the hamburger menu is currently only supported on Obsidian desktop (i.e. not mobile like iPad, tablet, etc.)

This workaround does not work for partial exports AFAIK.

**Comment by simonmeggle** [comment]
> This workaround does not work for partial exports AFAIK.

Yes, unfortunately. It exports the whole drawing. 
Partial export was really handy when it was working...

**Comment by JosephBrooksbank** [comment]
This does work in excalidraw.com. My workaround for now is to copy the whole file into excalidraw.com, select the section(s) I need to make images of, and copy them from the browser version. Dark mode is maintained

**Comment by tomaszchanek** [comment]
You may want to try "Export image..." with below option. Only selected elements are exported.

![image](https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/87475830/09c02ba3-daf2-4107-9ebc-5d36db3723fe)

**Comment by simonmeggle** [comment]
@tomaszchanek  That's what @zsviczian  recommended ;-)
